### DISCUSSION TITLE
Using spnego in a redhat server and windows ad

### DISCUSSION THREAD

**Discussion by eranbraun** [stnoonan/spnego-http-auth-nginx-module#114] [Q&A]
Hello everyone,
I am currently struggling to implement sso authentication with nginx and spnego in docker on a redhat host with active directory on windows server.
Till now I have configured the nginx to use the realm and the service principles. 
In Chrome I am receiving a 401 unauthorized error. I don't know a lot about kerberos, I don't know if it is a client issue but I would like to have some help to direct to problem to a solution.
Thank you.

**Comment by chaisacchu** [comment]
@eranbraun Above issues is resolved?, if Yes kindly help me to understand.

**Reply by owirtifo** [reply]
No
